From 96affbf39bb392f3fe4675d78bb58cdc45433441 Mon Sep 17 00:00:00 2001
From: CI Assistant <yossi.ilkanaev@intel.com>
Date: Fri, 22 Feb 2019 14:17:45 +0100
Subject: [PATCH] Merge pull request #496 in SW_UGW/linux from
 bugfix/DRVLIB_SW-1485-phy-not-working-when-linking-with-100m to master

* commit '607f3ef8bc4a6cc86424d80252469492107ae0b8':
  DRVLIB_SW-1485 - Switch 10/100M to use default LMAC instead of XGMAC
---
 drivers/net/ethernet/lantiq/switch-api/mac/mac_cfg.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/drivers/net/ethernet/lantiq/switch-api/mac/mac_cfg.c b/drivers/net/ethernet/lantiq/switch-api/mac/mac_cfg.c
index 48a7254db076..de1ee05ed41b 100644
--- a/drivers/net/ethernet/lantiq/switch-api/mac/mac_cfg.c
+++ b/drivers/net/ethernet/lantiq/switch-api/mac/mac_cfg.c
@@ -242,7 +242,6 @@ int mac_set_mii_if(void *pdev, u32 mii_mode)
 		gswss_set_fe_intf(pdev, LMAC_MII);
 	} else if (mii_mode == GSW_PORT_HW_GMII) {
 		gswss_set_1g_intf(pdev, XGMAC_GMII);
-		gswss_set_fe_intf(pdev, XGMAC_GMII);
 		gswss_set_2G5_intf(pdev, XGMAC_GMII);
 	} else if (mii_mode == 5) {	/* For Testing LMAC 1G only */
 		gswss_set_1g_intf(pdev, LMAC_GMII);
@@ -1099,7 +1098,7 @@ int mac_init(void *pdev)
 	/* Default enable flow control Rx and TX */
 	mac_set_flowctrl(pdev, 3);
 #endif
-	/* Configure MII for 100M, 10M, 1G and 2.5G to XGMAC by default
+	/* Configure MII for 1G and 2.5G to XGMAC by default
      * default value is LMAC
      */
 	mac_set_mii_if(pdev, GSW_PORT_HW_GMII);
